lib/modules/betteReddit.js

```diff
@@ -121,6 +121,12 @@ addModule('betteReddit', function(module, moduleID) {
 			type: 'boolean',
 			value: false,
 			description: 'Modify reddit\'s comment/link buttons ("perma-link source save...") such that they don\'t show up in searches. Disabled by default due to a slight performance impact.'
+		},
+		showHiddenSortOptions: {
+			type: 'boolean',
+			value: true,
```

Comment:
Not sure if this should be enabled by default. I'm imagining a bunch of reports of "q&a sort is showing up on non-q&a pages".